lib/std/os/windows/ntdll.zig

```diff
@@ -223,6 +223,10 @@ pub extern "ntdll" fn RtlWaitOnAddress(
     Timeout: ?*const LARGE_INTEGER,
 ) callconv(WINAPI) NTSTATUS;
 
+pub extern "NtDll" fn RtlUpcaseUnicodeChar(
```

Comment:
```suggestion
pub extern "ntdll" fn RtlUpcaseUnicodeChar(
```
3c2eddae5a4e89ec428a7b3b9f089c32f57b17af